### ISSUE TITLE
[Bug]: Lemmy build fails failed to parse manifest , invalid type: map, expected a string for key `package.description`

### ISSUE THREAD

**Issue by shodanx2** [LemmyNet/lemmy#4921]
### Requirements

- [X] Is this a bug report? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a single bug? Do not put multiple bugs in one issue.
- [X] Do you agree to follow the rules in our [Code of Conduct](https://join-lemmy.org/docs/code_of_conduct.html)?
- [X] Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.

### Summary

 Hi,
I am installing Lemmy on a Debian 11 VPS
I am following this tutorial

https://join-lemmy.org/docs/administration/from_scratch.html

I made it to the following step until I encountered an error

```
cargo build --release
error: failed to parse manifest at `/root/lemmy/Cargo.toml`

Caused by:
  invalid type: map, expected a string for key `package.description`
```

I have then investigate the contents of Cargo.toml

There is not the string `package.description` inside of it

```
cat Cargo.toml | grep package
[workspace.package]
[package]
```

```
cat Cargo.toml | grep description
description = "A link aggregator for the fediverse"
description.workspace = true
```

### Steps to Reproduce

1.Start with debian 11
2. follow tutorial https://join-lemmy.org/docs/administration/from_scratch.html
3. get error

### Technical Details

    Please post your log: sudo docker-compose logs > lemmy_log.out.
not using docker

    What OS are out trying to install lemmy on?
Debian 11

    Any browser console errors?
None

### Version

https://github.com/LemmyNet/lemmy.git lemmy

### Lemmy Instance URL

Not functionning, private instance

**Comment by Nothing4You** [comment]
It seems the instructions are outdated, as they're referring to Lemmy 0.18.5 still.
Instead of the `git checkout 0.18.5` you should use `git checkout 0.18.5` instead.

That's not the underlying problem though.

Can you please provide the output of `cargo version`?

**Comment by Nothing4You** [comment]
Either way, I've just successfully built Lemmy 0.18.5 on Debian 11 by following the instructions step by step. 
The only thing I had to add was installing `build-essential` via apt to get the build time dependencies installed, but that is long past cargo parsing errors.

`package.description` would not exist in `Cargo.toml` as verbatim string, this is just a reference for where it's accessed.
This is a reference to the `description` field in the `[package]` section.
